index.html

```diff
@@ -1831,322 +1831,291 @@ <h2>Proof</h2>
   <section class="normative">
     <h1>Core Representations</h1>
 
-      <p>
-A <a>DID document</a> MUST be a single JSON object conforming to [[!RFC8259]].
-Many of the concepts in this document were introduced by example using the
-JSON-LD syntax, a format for mapping JSON data into the RDF semantic graph
-model, as defined by [[!JSON-LD]]. This section formalizes how the data model
-(described in Sections <a href="#data-model"></a> and
-<a href="#did-documents"></a>) is realized in JSON-LD.
-      </p>
-      <p>
-Although syntactic mappings are provided for JSON and JSON-LD only, applications
-and services can use any other data representation syntax, such as JXD (JSON XDI
-Data, a serialization format for the
-<a href="http://docs.oasis-open.org/xdi/xdi-core/v1.0/csd01/xdi-core-v1.0-csd01.xml">XDI graph model</a>),
-XML, YAML, or CBOR, that is capable of expressing the data model.
-      </p>
-
-    <section>
-      <h2>
-Pure JSON
-      </h2>
-
-        <p>
-The data model, as described in Section <a href="#data-model"></a>, can be
-encoded in Javascript Object Notation (JSON) [[RFC8259]] by mapping property
-values to JSON types as follows:
-        </p>
-
-        <ul>
-          <li>
-Numeric values representable as IEEE754 SHOULD be represented as a Number type.
-          </li>
-          <li>
-Boolean values SHOULD be represented as a Boolean type.
-          </li>
-          <li>
-Sequence value SHOULD be represented as an Array type.
-          </li>
-          <li>
-Unordered sets of values SHOULD be represented as an Array type.
-          </li>
-          <li>
-Sets of properties SHOULD be represented as an Object type.
-          </li>
-          <li>
-Empty values SHOULD be represented as a null value.
-          </li>
-          <li>
-Other values MUST be represented as a String type.
-          </li>
-        </ul>
-    </section>
+    <p>
+All concrete representations of a <a>DID document</a> MUST be serialized using a 
```

Comment:
How do we test this normative requirement in the test suite?

Reply:
This is exactly what the test suite is for. We give a producer a data model and tell it to produce a document which we need to be able to parse into an unambiguously equal data model.

Reply:
> We give a producer a data model

How exactly are we doing that? What does the input file look like?